app/Domain/Integrations/Integration.php

```diff
@@ -27,7 +27,7 @@ public function __construct(
         $this->urls = [];
     }
 
-    public function withContacts(Contact ...$contacts): self
+    public function withContacts(Contact...$contacts): self
```

Comment:
```suggestion
    public function withContacts(Contact ...$contacts): self
```
I would expect the formatting to look like this